packages/connect-web/webpack/prod.webpack.config.ts

```diff
@@ -73,6 +73,7 @@ const config: webpack.Configuration = {
         }),
     ],
     optimization: {
+        minimize: false,
```

Comment:
Is this just for development or nay reason to have it in prod?

Reply:
It's to keep comments, because of inlined content script code in webextension. Since I moved some files around I needed to change the config in this package as well otherwise the comment that marks the place to put the content script would get dropped

Reply:
I see.